src/test/datascience/notebook/interruptRestart.vscode.test.ts

```diff
@@ -168,7 +168,7 @@ suite('DataScience - VSCode Notebook - Restart/Interrupt/Cancel/Errors (slow)',
             throw new Error('Kernel not available');
         }
         const waitForKernelToRestart = createEventHandler(kernel, 'onRestarted', disposables);
-        commands.executeCommand('jupyter.notebookeditor.restartkernel').then(noop, noop);
+        await commands.executeCommand('jupyter.notebookeditor.restartkernel').then(noop, noop);
```

Comment:
I didn't expect this to fix it, but I ran it 10 times and always got a pass. (It was flakey before)

Reply:
Hmm given the code it seems a bit funny. Given how it's waiting below seems like it could have been just fire and forget like this. Probably doesn't need the then statement now, but works as is.